pom.xml

```diff
@@ -319,6 +319,13 @@
                             <arg>-Xlint:all</arg>
                             <!-- TODO <arg>-Werror</arg> -->
                         </compilerArgs>
+                        <annotationProcessorPaths>
+                            <annotationProcessorPath>
+                                <groupId>com.google.auto.service</groupId>
```

Comment:
is autoservice being used anywhere?

Reply:
I can try to move it on jetcd-core but it does not hurt :)